src/view.rs

```diff
@@ -136,21 +138,23 @@ pub struct ListBuildContext {
 }
 
 /// A type alias for the closure.
-type ListBuilderClosure<T> = dyn Fn(&ListBuildContext) -> (T, u16);
+type ListBuilderClosure<'render, T> = dyn Fn(&ListBuildContext) -> (T, u16) + 'render;
 
 /// The builder to for constructing list elements in a `ListView<T>`
-pub struct ListBuilder<T> {
-    closure: Box<ListBuilderClosure<T>>,
+pub struct ListBuilder<'render, T> {
+    closure: Box<ListBuilderClosure<'render, T>>,
+    _phantom: PhantomData<&'render T>,
```

Comment:
do we need this? We already use the lifetime in the closure.

Reply:
I needed it at some point when I was getting things working, but I removed it and all the tests still pass! It seem's totally unnecessary now.